src/app/components/trip-information/trip-information.component.scss

```diff
@@ -0,0 +1,23 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Yeseva+One&display=swap');
```

Comment:
No need to import fonts, they are implemented as a link in index.html file from google fonts.